src/layout/FileUpload/types.d.ts

```diff
@@ -8,6 +8,7 @@ export interface ILayoutCompFileUploadBase<T extends Extract<ComponentTypes, 'Fi
   displayMode: 'simple' | 'list';
   hasCustomFileEndings?: boolean;
   validFileEndings?: string[] | string;
+  alertOnDelete?: boolean;
```

Comment:
```suggestion
  alertOnDelete?: ExprVal.Boolean;
```

In case you're changing this to be an expression. (Need to import ExprVal as well, so don't just commit this suggestion)